packages/babel-plugin-jest-deadlines/src/__tests__/deadlinesPlugin.test.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+/* eslint-disable sort-keys */
+
+import pluginTester from 'babel-plugin-tester';
+import babelPluginJestDeadlines from '..';
+
+pluginTester({
+  plugin: babelPluginJestDeadlines,
+  pluginName: 'babel-plugin-jest-deadlines',
+  tests: {
+    'bare function call in statement context': {
+      // language=JavaScript
```

Comment:
what does this comment do?

Reply:
It triggers Intellij (i.e. webstorm) to treat the string literal as if it was real code, so IDE features like completion and formatting work inside. Not critical at all, just a copy-paste from the other project.

![image](https://user-images.githubusercontent.com/328180/101081773-6b884980-35a2-11eb-9b64-c2853334da7a.png)

Reply:
fancy!

(I've used intellij since 2012 and didn't know that 😛)